server/storage/wal/wal_test.go

```diff
@@ -96,6 +96,60 @@ func TestNew(t *testing.T) {
 	}
 }
 
+func TestCreateNewWALFile(t *testing.T) {
+	tests := []struct {
+		name     string
+		fileType interface{}
+		truncate bool
+	}{
+		{
+			name:     "creating standard file should succeed",
+			fileType: &os.File{},
+		},
+		{
+			name:     "creating locked file should succeed",
+			fileType: &fileutil.LockedFile{},
+		},
+		{
+			name:     "creating standard file with truncate should succeed",
+			fileType: &os.File{},
+			truncate: true,
+		},
+		{
+			name:     "creating locked file with truncate should succeed",
+			fileType: &fileutil.LockedFile{},
+			truncate: true,
+		},
```

Comment:
can you also add two cases to verify the following two cases?
- truncate:`true`, it should truncate the file content if present
- truncate: `false`: it shouldn't truncate the file content if present

Reply:
Added test cases to verify truncate behavior